pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/dao/impl/PatientRecordDAOImpl.java

```diff
@@ -10,29 +32,71 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Repository;
 
-import gov.ca.emsa.pulse.broker.dao.OrganizationDAO;
-import gov.ca.emsa.pulse.broker.dao.PatientRecordDAO;
-import gov.ca.emsa.pulse.broker.dto.PatientRecordDTO;
-import gov.ca.emsa.pulse.broker.entity.PatientRecordEntity;
-
 @Repository
 public class PatientRecordDAOImpl extends BaseDAOImpl implements PatientRecordDAO {
 	private static final Logger logger = LogManager.getLogger(PatientRecordDAOImpl.class);
 	@Autowired OrganizationDAO orgDao;
+	@Autowired PatientRecordNameDAO nameDao;
```

Comment:
It's not clear to me why the methods createOrgMap and updateOrgMap were moved from PatientDAOImpl to PatientRecordDAOImpl. I think they didn't need to be?